components/brave_wallet_ui/panel/async/wallet_panel_async_handler.ts

```diff
@@ -501,6 +514,26 @@ handler.on(PanelActions.signAllTransactionsProcessed.type, async (store: Store,
   panelHandler.closeUI()
 })
 
+handler.on(PanelActions.signMessageError.type, async (store: Store) => {
+  store.dispatch(PanelActions.navigateTo('siweError'))
+  const { panelHandler } = getWalletPanelApiProxy()
+  panelHandler.showUI()
+})
```

Comment:
Can we simply check for errors and show the error panel when an error is detected instead of manually navigating?

Reply:
Removed navigation